userspace/libscap/linux/scap_procs.c

```diff
@@ -1386,7 +1386,7 @@ int32_t scap_linux_get_fdlist(struct scap_platform* platform, struct scap_thread
 	struct scap_linux_platform* linux_platform = (struct scap_linux_platform*)platform;
 
 	// We collect file descriptors only for the main thread
-	snprintf(proc_dir, sizeof(proc_dir), "%s/proc/%d/", scap_get_host_root(), tinfo->pid);
+	snprintf(proc_dir, sizeof(proc_dir), "%s/proc/%lu/", scap_get_host_root(), tinfo->pid);
```

Comment:
so in scap we use `uint64_t` but in sinsp, we use `int64_t` :confused: Wow

Reply:
I know ..